app/models/course/assessment/submission.rb

```diff
@@ -237,4 +278,49 @@ def phantom_submission_email_enabled?
     Course::Settings::AssessmentsComponent.
       email_enabled?(assessment.tab.category, :new_phantom_submission)
   end
+
+  def csv_empty
```

Comment:
Probably should shift all these methods to service, since it will be only used for download service and we prefer to only keep the methods that are used globally in the model.

Reply:
Ok, I think I just shift the ```download_statistics``` and ```download_statistics_header``` to service as well?

Reply:
👍 yep, thanks! and also include the helper only in the service